3scaleAdapter/pkg/threescale/threescale.go

```diff
@@ -202,6 +213,12 @@ func (s *Threescale) backendClientBuilder(backendURL string) (*backendC.ThreeSca
 	return backendC.NewThreeScale(be, s.client), nil
 }
 
+// errorWrapper is a utility function to reduce repeated code when the adapter
+// needs to report an error. Takes an integer and c
+func (s *Threescale) errorWrapper() {
+
+}
```

Comment:
this seems to have been accidentally left empty?

Reply:
yep should have been removed. Thanks